### PR TITLE
render audio fields

### PR DESCRIPTION
**Pull request by luxio**

### CODE REVIEW

File: media.js [hunk 1]
```diff
@@ -65,6 +65,14 @@ function theme_media_file_rendered(variables) {
               class: 'media-video'
             }
           });
+        } else if ((/audio\//).test(item.filemime)) {
+          return theme('audio', {
+            path: drupalgap_image_path(item.uri),
+            attributes: {
+              controls: '',
+              class: 'media-video'
```
Comment (signalpoint) [inline media.js]: @luxio Should this class be `media-audio` ?

Comment (luxio) [inline media.js]: Yes. You are right.

Comment (luxio) [inline media.js]: I will fix this and send a new one.

**Comment by signalpoint** [comment]
@luxio Nicely done, thank you!

**Pull request merged**